src/helpers/pagination.ts

```diff
@@ -0,0 +1,30 @@
+interface PaginateProps {
+  items: unknown[];
+  pageNumber: number;
+  pageSize: number;
+}
+
+const DEFAULT_PAGE_SIZE = 10;
+
+const paginate = ({ items, pageNumber, pageSize }: PaginateProps) => {
+  const startIndex = (pageNumber - 1) * pageSize;
+  return items.slice(startIndex, startIndex + pageSize);
```

Comment:
Should add a line here?

Reply:
Fixed in https://github.com/nimblehq/til-web/pull/18/commits/48d02e70933d005b05419a804c13c51635f8f8fd